### ISSUE TITLE
mmextract does not work with integers

### ISSUE THREAD

**Issue by Kirk3gaard** [KasperSkytte/mmgenome2#24]
When plotting something that has integer values e.g. length the selection tool breaks when using mmextract.  e.g.

```
sel<-data.frame(length = c(3186796.968, 3139304.135, -53787.213, -42302.204),
           cov_I_01 = c(198.8, 279.547, 280.235, 223.964))

mmplot(mm = d_01,locator = F,selection = sel,
       x = "length",
       y = "cov_I_01"
       ) 
d_01_subset<-mmextract(mm = d_01,selection = sel)

```
Gives the error:

`Error in sp::point.in.polygon(point.x = mms[[colnames(selection)[1]]], : REAL() can only be applied to a 'numeric', not a 'integer'`

**Comment by jronwalker** [comment]
I am experiencing this exact same issue on a specific dataset. Other data sets are not having the same issue and I have copied and pasted the same commands from the dataset that works.

**Comment by KasperSkytte** [comment]
Fixed in 2d39bec7d2f8ea0cd677d1a1a2d0d84461c5061e
